### PR TITLE
Pending Collaborations

### PR DESCRIPTION
**Pull request by marek-vysoky**
Adding method GetPendingCollaborationAsync with comments, adding unit test GetPendingCollaboration_ValidResponse_ValidEntries

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @marek-vysoky has signed the CLA. Thanks for the pull request!

**Comment by coolboy** [comment]
@marek-vysoky please resolve the conflict here

**Comment by marek-vysoky** [comment]
I've merged master to this branch, resolved the conflict and push the changes

File: Box.V2/Managers/BoxCollaborationsManager.cs [hunk 1]
```diff
@@ -105,5 +105,20 @@ public BoxCollaborationsManager(IBoxConfig config, IBoxService service, IBoxConv
...
         }
+        /// <summary>
+        /// Used to retrieve all pending collaboration invites for this user.
+        /// </summary>
+        /// <param name="fields">Attribute(s) to include in the response</param>
+        /// <returns>A collection of pending collaboration objects are returned. If the user has no pending collaborations, the collection will be empty.</returns>
+        public async Task<BoxCollection<BoxCollaboration>> GetPendingCollaborationAsync(List<string> fields = null)
+        {
+           
+            BoxRequest request = new BoxRequest(_config.CollaborationsEndpointUri, null)
```
Comment (marek-vysoky) [inline Box.V2/Managers/BoxCollaborationsManager.cs]: I did it as it is in GetCollaborationAsync

**Pull request merged**